### ISSUE TITLE
Add VBO options that Drupal's VBO has

### ISSUE THREAD

**Issue by rudiedirkx** [backdrop/backdrop-issues#802]
Specifically:

* Show separate action buttons instead of dropdown + submit
* Override labels (for includes/selected actions)

**Comment by rudiedirkx** [comment]
PR: https://github.com/backdrop/backdrop/pull/768

**Comment by quicksketch** [comment]
Hi @rudiedirkx! Thanks for filing this issue. Maintaining compatibility with D7 options seems like a good idea. I'm on the fence about whether this should be in core or separated into a contrib module. I think I personally would be okay with them in core, but I'd like to hear what others say.

In any case, this is a clear feature expansion and it needs tests to ensure that it works properly. The current approach to VBO is tested by the Node and Comment management interfaces. I'd recommend making a dedicated test class in views.test for VBO functionality.

**Comment by rudiedirkx** [comment]
I don't think it can be contrib, because it adds views options and those can only be added (or at least exported) by the handler itself. The VBO field has to define all options. At least that's how D7 Views works, and I'm guessing that hasn't changed.

I'll look into the tests, maybe.

**Comment by quicksketch** [comment]
After thinking about it for a while, I think it would be a good addition to the core functionality, so I'm all for it. However, we definitely need tests to cover the new functionality. I've close the PR for the time being until tests can be written.

**Comment by klonos** [comment]
> - Show separate action buttons instead of dropdown + submit

I am wondering whether it'd be worth replacing the dropdown + submit with a dropbutton + confirmation dialog where needed. In which case, this set of options for "Display operations" would become:

- ( ) Single drop-button for all operations
- ( ) Separate buttons for each operation

Other things to consider:

- Allow setting the default operation
- Allow setting the order of the operations
- If rendered as separate buttons, the default should be indicated with blue color
- If the operation is destructive, show the button as red/danger
